docker/web/Dockerfile

```diff
@@ -56,7 +56,7 @@ ARG APP_ENV=prod
 COPY ./composer.json ./composer.lock /app/
 RUN mkdir -p var/cache var/sessions public/tmp \
     # Install dependencies
-    && composer install --prefer-dist --no-dev --no-autoloader --no-scripts --no-progress --no-suggest \
```

Comment:
Pourquoi est ce que les suggestions doivent maintenant êtres installées. Si des paquets dans les suggests sont requis, il est préférable de les mettre dans les dépendances explicites

Reply:
J’avais le commentaire `You are using the deprecated option "–no-suggest". It has no effect and will break in Composer 3`.
Mais je ne sais pas si on va passer un jour à composer 3 et je n’ai pas trouvé d’équivalent.

Reply:
Ok, makes sense. Merci :+1: